spec/models/candidate_interface/contact_details_form_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe CandidateInterface::ContactDetailsForm, type: :model do
+  let(:data) do
+    {
+      phone_number: '07700 900 982',
+    }
+  end
+
+  describe '.build_from_application' do
+    it 'creates an object based on the provided ApplicationForm' do
```

Comment:
`creates an object...` sounds a bit generic, we should be a bit more descriptive,
maybe `puts phone data into contact_details` ... I don't know, you got the point :)

Reply:
I'm not sure what to change it to. Only could think of `creates a ContactDetailsForm object based on the provided ApplicationForm` which is more specific but possibly not significant enough.